Benchmark-Models/Laske_PLOSComputBiol2019/parameters_Laske_PLOSComputBiol2019.tsv

```diff
@@ -0,0 +1,14 @@
+parameterId	parameterName	parameterScale	lowerBound	upperBound	nominalValue	estimate
+k_imp	k_{Imp}	log10	1E-10	10000000000	0.296	1
+k_syn_R_M	R^{Syn}_{R_M}	log10	1E-10	10000000000	30600	1
```

Comment:
are these parameter boundaries given in the paper?? this seems to be HUGE

Reply:
They are not given in the paper. We used the Scipy optimizer in the beginning and the parameter bounds were hit frequently. Therefore, they were enlarged in case it was required for reliable optimization. After that we used Fides; the bounds were left untouched. They could probably be reduced though.

Reply:
if you have an idea of reasonable biological ranges, maybe you could adapt them? what @dilpath think?

Reply:
I am not entirely sure what's biologically reasonable. I can only assume what might be reasonable from the parameter estimations and sampling that we performed. It would probably also require testing the parameter estimation again, as I am not sure what bounds are needed for reliable optimization.

Reply:
oki, then we just keep it as it is 😄